src/arma3-unix-launcher/tablewidgetcheckboxitem.h

```diff
@@ -0,0 +1,22 @@
+#ifndef TABLEWIDGETCHECKBOXITEM_H
+#define TABLEWIDGETCHECKBOXITEM_H
+
+#include <QTableWidgetItem>
+
+class TableWidgetCheckboxItem : public QTableWidgetItem
+{
+public:
+    explicit TableWidgetCheckboxItem(bool enabled);
+
+    virtual bool operator<(const QTableWidgetItem& other) const override {
```

Comment:
You don't need `virtual` keyword when overriding, but that's not a problem - I'll correct it after merging.